### PR TITLE
docs: add documentation

### PR DESCRIPTION
**Pull request by paschalidi**
Closes the [ticket](https://jira.dhis2.org/browse/DHIS2-10161) 

👋 @geethaalwan check this out and please let me know your thoughts. :)

### CODE REVIEW

**Review by karolinelien — COMMENTED** [review]
@paschalidi I added a commit with some spelling/clarification edits.

File: src/commonmark/en/content/user/using-the-capture-app.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ Note that, rules can be configured in the maintenance app.
 
 To see a rule being executed while enrolling a tracked entity instance you will have to take the following steps. 
 
-1. Configure a rule in the maintenance app. For the example bellow we configured a rule that throws a warning when the date of birth is less than a year.
+1. Configure a rule in the maintenance app. For the example below we configured a rule that throws a warning when the date of birth is less than a year.
```
Comment (paschalidi) [inline src/commonmark/en/content/user/using-the-capture-app.md:242]:
```suggestion
1. Configure a rule in the maintenance app. For the example below we configured a rule that throws a warning when the date of birth is less than a year!
```

File: src/commonmark/en/content/user/using-the-capture-app.md [hunk 2]
```diff
@@ -758,3 +758,78 @@ For performance reasons the Capture app caches metadata in the client browser. W
...
 
 3. The exception to the two rules above is option sets. Option sets have their own version property, i.e. increasing the option set version should ensure the option set metadata are propagated to the clients.
+
+
+## Enrollment dashboard
+
+### Reaching the enrollment dashboard via url
+
+You reach the enrollment dashboard either by typing in the address bar of your browser or using the user interface of the capture app.
+In this section we are focusing on the first use-case, where you type or paste the url you want to access in the Address bar.
```
Comment (paschalidi) [inline src/commonmark/en/content/user/using-the-capture-app.md:768]:
```suggestion
You reach the enrollment dashboard either by typing in the address bar of your browser or using the user interface of the capture app.
In this section we are focusing on the first use-case, where you type or paste the url you want to access in the Address bar....
```

File: src/commonmark/en/content/user/using-the-capture-app.md [hunk 3]
```diff
@@ -758,3 +758,78 @@ For performance reasons the Capture app caches metadata in the client browser. W
...
 
 3. The exception to the two rules above is option sets. Option sets have their own version property, i.e. increasing the option set version should ensure the option set metadata are propagated to the clients.
+
+
+## Enrollment dashboard
+
+### Reaching the enrollment dashboard via url
+
+You reach the enrollment dashboard either by typing in the address bar of your browser or using the user interface of the capture app.
+In this section we are focusing on the first use-case, where you type or paste the url you want to access in the Address bar....
```
Comment (paschalidi) [inline src/commonmark/en/content/user/using-the-capture-app.md]:
```suggestion
You reach the enrollment dashboard either by typing in the address bar of your browser or using the user interface of the capture app.
In this section we are focusing on the first use-case, where you type or paste the url you want to access in the Address bar.
```

File: src/commonmark/en/content/user/using-the-capture-app.md (hunk 1 above)
Comment (paschalidi) [reply src/commonmark/en/content/user/using-the-capture-app.md:242]: Well this is not alright

**Review by karolinelien — APPROVED** [review]

**Review by geethaalwan — APPROVED** [review]

**Review by Philip-Larsen-Donnelly — APPROVED** [review]

**Pull request merged**